src/lib/config/ConfigFileManager.ts

```diff
@@ -0,0 +1,51 @@
+import path from "path";
+import os from "os";
+import fs from "fs";
+
+export class ConfigFileManager {
+  private folderPath: string;
+  private configFilePath: string;
+
+  constructor(baseFolder: string = ".genlayer/", configFileName: string = "genlayer-config.json") {
+    this.folderPath = path.resolve(os.homedir(), baseFolder);
+    this.configFilePath = path.resolve(this.folderPath, configFileName);
+    this.ensureFolderExists();
+    this.ensureConfigFileExists();
+  }
+
+  private ensureFolderExists(): void {
+    if (!fs.existsSync(this.folderPath)) {
+      fs.mkdirSync(this.folderPath, { recursive: true });
+    }
+  }
+
+  private ensureConfigFileExists(): void {
+    if (!fs.existsSync(this.configFilePath)) {
+      fs.writeFileSync(this.configFilePath, JSON.stringify({}, null, 2));
+    }
+  }
+
+  getFolderPath(): string {
```

Comment:
@epsjunior what are these two functions used for?

Reply:
> @epsjunior what are these two functions used for?

@cristiam86 which functions? ensureConfigFileExists and ensureFolderExists?

Reply:
It's just a utility function to get the path of our configuration folder. It's not being used right now but it could be used in future features or commands